packages/flutter/test/widgets/box_decoration_test.dart

```diff
@@ -61,20 +66,20 @@ Future<void> main() async {
     expect(tester.binding.hasScheduledFrame, isTrue);
     await tester.pump();
     expect(tester.binding.hasScheduledFrame, isFalse);
+    // Evicts an entry from the image cache.
+    imageProvider.evict();
```

Comment:
Will it be more readable to invoke 
`addTearDown(imageProvider.evict);` immediately after creating imageProvider?

Applies to other cases too.

Reply:
Thanks for review!!!

Yes indeed it would be more readable will update this and update here, thank you

Reply:
@polina-c As per suggestion, have updated code with `addTearDown(imageProvider.evict);` 😄